app/scripts/actions/router.js

```diff
@@ -0,0 +1,3 @@
+import {push, replace, go, goBack, goForward} from 'react-router-redux'
+
+export {push, replace, go, goBack, goForward}
```

Comment:
This feels superfluous, lets replace the imports of these with direct imports to `react-router-redux`